File: Trabalho_Final/TrabalhoFinal/src/servidor/Conection.java
package servidor;

import java.io.IOException;
import java.net.DatagramSocket;

public class Conection {

    public static void main(String[] args) throws IOException {
        DatagramSocket socket = null;
        UDPServe servidor = new UDPServe(socket);

        while (true) {
            servidor.sendRequest();
            servidor.sendResponse();
        }
    }
}

File: Trabalho_Final/TrabalhoFinal/src/servidorJS/Conection.js
const UDPServe = require('./UDPServer');
const udp = require('dgram');

class Conection {
    falha = 1;

    main() {
        const socket = udp.createSocket('udp4');
        const porta = 1515;
        const servidor = new UDPServe(socket, porta);

        socket.on('message', (msg, rinfo) => this.falha++ % 3 && servidor.getRequest(msg, rinfo));

        socket.bind(porta);
    }
}

new Conection().main();